dotcom-rendering/src/web/browser/sentryLoader/sentry.ts

```diff
@@ -35,7 +35,7 @@ const {
 
 Sentry.init({
 	ignoreErrors,
-	whitelistUrls,
+	allowUrls,
```

Comment:
Sentry has actually deprecated `whitelistUrls` in favour of [`allowUrls`](https://docs.sentry.io/platforms/javascript/guides/remix/configuration/options/#allow-urls).